src/pages/Options/Home/MarketsTable/MarketsTable.tsx

```diff
@@ -231,7 +231,11 @@ const PhaseDiv = styled.div<{ phase: Phase; isLabel: boolean }>`
 			css`
 				background-color: #c5d5ff;
 			`}
-
+		${(props) =>
+			props.phase === 'expiry' &&
+			css`
+				background-color: #f5607066;
```

Comment:
is this correct?

Reply:
should we be using a variable from the theme colors?

Reply:
@evgenyboxer what do you mean?

Reply:
oh sorry I thought the color was invalid, but it is!

Reply:
@dvd-schwrtz agreed. Not I saw the previous block was using hardcoded colours so I did the same

Reply:
@evgenyboxer yeah I added a 40% opacity on it, which makes it look weird I agree.